src/pages/MerchantVoucherDashboardV2/MerchantVoucherDashboardV2.tsx

```diff
@@ -52,7 +52,13 @@ const MerchantVoucherDashboardV2 = () => {
     return <ErrorPage menuOpen={false} />;
   }
 
-  return <VoucherDashboard giftCards={giftCards} organizationName={seller.name} />;
+  return (
+    <VoucherDashboard
+      fetchData={fetchData}
```

Comment:
probs don't need to pass down the entire fetch (which incl seller and gift card fetch), but it's easier to do it this way / it's not too much more latency hopefully. open to just passing down gift card fetch as well though